src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,57 @@
 package core.basesyntax;
 
+
+import java.io.*;
+
 public class WorkWithFile {
+    int buySum = 0;
+    int supplySum = 0;
     public void getStatistic(String fromFileName, String toFileName) {
+        readFromFile(fromFileName);
+        calculateReport();
+        writeToFile(toFileName);
+        cleanUp();
+    }
```

Comment:
The method 'getStatistic' is doing too much. It's handling file reading, report calculation, file writing, and cleanup. Consider breaking this method into smaller ones, each handling a single responsibility.